spec/amp-email-format.md

```diff
@@ -1,3 +1,11 @@
+---
+$title: AMP for Email specification
+author: fstanis
+$order: 3
+formats:
+  - email
+---
```

Comment:
Should this be here? I don't see this table in the other specs.

Reply:
yes, it's needed for amp.dev's format filtering to work.